packages/site/cms-site/src/blocks/helpers/VideoPreviewImage.tsx

```diff
@@ -9,15 +9,14 @@ export interface VideoPreviewImageProps {
     aspectRatio: string;
     sizes?: string;
     fill?: boolean;
+    icon?: React.ReactElement;
 }
 
-export const VideoPreviewImage = ({ onPlay, image, aspectRatio, sizes = "100vw", fill }: VideoPreviewImageProps) => {
+export const VideoPreviewImage = ({ onPlay, image, aspectRatio, sizes = "100vw", fill, icon }: VideoPreviewImageProps) => {
```

Comment:
Optional: You could also use a default value here (`icon = <PlayIcon />`) and remove the if in the render